streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -141,6 +139,8 @@ public void tearDown() throws IOException {
 
     @Test
     public void testRegexMatchesTopicsAWhenCreated() throws Exception {
+        final String outputTopic = createTopic(topicSuffixGenerator.incrementAndGet());
```

Comment:
I think this the counter, we can keep this in `setUp()` method and make `outputTopic` a class member?

Reply:
ack, that's what I get for moving to fast.